src/main/constraints/107_default_instant_launches.sql

```diff
@@ -0,0 +1,10 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE default_instant_launches
+    ADD CONSTRAINT default_instant_launches_added_by_fkey
+    FOREIGN KEY (added_by)
+    REFERENCES users(id);
+
+ALTER TABLE default_instant_launches
+    ADD CONSTRAINT default_instant_launches_id_version_unique
+    UNIQUE (id, version);
```

Comment:
If I'm understanding things correctly, it seems to me that this uniqueness constraint should be applied only to the `version` column.

Reply:
Yeah, I think this won't do much as is, since `id` is already unique. It will create an index on `(id, version)`, but particularly with the unique column first I'm not sure it's ary more useful than the primary key index on `id`.

Reply:
I removed the id field from the constraint.